posts/How-to-add-blog-post/index.qmd

```diff
@@ -0,0 +1,253 @@
+---
+title: "How to add a blog post using Quarto"
```

Comment:
```suggestion
title: "How to create a blog post in the Open Science Community Saudi Arabia using Quarto?"
```

Reply:
alright, noted, i'll make the necessary changes